families-api/app/router.py

```diff
@@ -254,7 +256,21 @@ class GeographyDocumentCount(SQLModel):
 )
 def docs_by_geo(
     session: Session = Depends(get_session),
+    corpus_import_ids: list[str] = Query(
+        default=[],
+        alias="corpus.import_id",
+    ),
+    document_statuses: list[FamilyDocumentStatus] = Query(
+        default=[],
+        alias="document.document_status",
```

Comment:
To be a complete pedant - in the `Family` model - [this is represented as `documents`](https://github.com/climatepolicyradar/navigator-backend/blob/main/families-api/app/models.py#L379-L384)...


```suggestion
        alias="documents.document_status",
```